components/Proposals/ProposalList.tsx

```diff
@@ -32,13 +35,49 @@ export default function ProposalList() {
 
   return (
     <Stack>
-      {programVersion !== null && (
-        <NativeSelect
-          label="Program version"
-          data={programVersions}
-          value={AUTOCRAT_VERSIONS.indexOf(programVersion!)}
-          onChange={(e) => setProgramVersion(Number(e.target.value))}
-        />
+      <Group justify="space-between">
+        {programVersion !== null && (
+          <>
+          <Menu>
+            <Menu.Target>
+              <Button variant="secondary"><IconSwitchVertical strokeWidth={0.85} /></Button>
+            </Menu.Target>
+            <Menu.Dropdown>
+              <Stack p="md" align="center">
+                <NativeSelect
+                  label="Program version"
+                  // NOTE: A slice for removing this from render given devnet
+                  data={Array.prototype.slice.call(programVersions, network === 'devnet' ? 0 : 1)}
```

Comment:
I think you might wanna just do `programVersions.slice( network === 'devnet' ? 0 : 1)` unless there is something I am missing about doing it this way.

Reply:
wow that's interesting. Alright so be it. 🤷